app.py

```diff
@@ -112,12 +112,18 @@ def experience():
 
 
 @app.route('/resume/education', methods=['GET', 'POST'])
-def education():
+@app.route('/resume/education/<int:index>', methods=['GET'])
+def education(index=None):
     '''
     Handles education requests
     '''
     if request.method == 'GET':
-        return jsonify({})
+        if index is not None:
+            if index < 0 or index >= len(data['education']):
+                return jsonify({'error': 'Index out of range'}), 404
```

Comment:
nit: it's usually better to just say Education not found and not make it overly detailed for potential attackers

Reply:
updated